docs/src/app/docs/get-started/spring-boot/page.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Spring Boot with LaunchFlow
+springboot:
+  metadata:
+    title: Spring Boot with LaunchFlow
+    description: Deploy Spring Boot to AWS / GCP with LaunchFlow
+---
+
+{% gettingStartedSelector  %}
+
+{% gettingStartedSection cloudProvider="AWS" runtime="ECS Fargate" %}
+
+Deploy a Spring Boot application to AWS Fargate with LaunchFlow.
+
+{% callout type="note" %}
+
+View the source code for this guide in our [examples repo](https://github.com/launchflow/launchflow-examples/tree/main/spring-boot-get-started/aws/ecs-fargate).
+
+{% /callout %}
+
+{% /gettingStartedSection %}
+
+{% gettingStartedSection cloudProvider="GCP" runtime="Cloud Run" %}
+
+Deploy a Spring Boot application to GCP's serverless runtime Cloud Run with LaunchFlow.
+
+{% callout type="note" %}
+
+View the source code for this guide in our [examples repo](https://github.com/launchflow/launchflow-examples/tree/main/spring-boot-get-started/gcp/cloud-run).
+
+{% /callout %}
+
+{% /gettingStartedSection %}
+
+{% gettingStartedSection cloudProvider="GCP" runtime="Compute Engine" %}
+
+Deploy a Spring Boot application to GCP Compute Engine VMs with LaunchFlow.
+
+{% callout type="note" %}
+
+View the source code for this guide in our [examples repo](https://github.com/launchflow/launchflow-examples/tree/main/spring-boot-get-started/gcp/compute-engine).
+
+{% /callout %}
+
+{% /gettingStartedSection %}
+
+{% gettingStartedSection cloudProvider="GCP" runtime="Kubernetes" %}
+
+Deploy a Spring Boot application to Kubernetes running on GKE with LaunchFlow.
+
+{% callout type="note" %}
+
+View the source code for this guide in our [examples repo](https://github.com/launchflow/launchflow-examples/tree/main/spring-boot-get-started/gcp/gke).
+
+{% /callout %}
+
+{% /gettingStartedSection %}
+
+
+## 0. Set up your Spring Boot Project
+
+If you already have a Spring Boot Project you can [skip to step #1](#1-initialize-launch-flow).
+
+---
+
+Create a new Spring Boot Application
+
+```bash
+spring init --dependencies=web launchflow-springboot
+cd launchflow-springboot
+```
+
+---
+
+Update `src/main/java/com/example/demo/DemoApplication.java` to include a simple REST controller:
+
+```java
```

Comment:
You can update https://github.com/launchflow/launchflow/blob/ff41b8eaa2468082a49e5f4b370d73e345a6af1f/docs/src/components/Fence.tsx#L9 to add support for java highlighting, I think you can just copy the docker one but s/docker/java